test/Dapr.PluggableComponents.Proxies/Protos/dapr/proto/components/v1/bindings.proto

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package dapr.proto.components.v1;
+
+import "dapr/proto/components/v1/common.proto";
+
+option csharp_namespace = "Dapr.PluggableComponents.Proxies.Grpc.v1";
+option go_package = "github.com/dapr/dapr/pkg/proto/components/v1;components";
+
```

Comment:
Nit: I am not sure how self-contained we want this package to be but I wonder if it wouldn't be better to have a step that downloads the .protos from a particular/tagged Dapr version.

The [Java SDK takes this approach](https://github.com/dapr/java-sdk/blob/master/sdk-autogen/pom.xml#L59-L104) and I would argue it has quite a few advantages.  It keeps Dapr repo as the authoritative source, decreases repetition and decreases the chances of code getting out of sync. 

The [python SDK takes a similar "copy protos" approach](https://github.com/dapr/python-sdk/blob/master/tools/regen_grpcclient.sh) but it has a tool to automate (and reduce chances of error) for this process.

Reply:
Generally agree.  I'd rather the duplication wasn't (immediately) necessary, even within this single repo.  However, I found that the `.proto` generation tools have certain expectations about relative location as well as issues with conflicting namespace names that made it simply more expedient to duplicate and modify (for now).  I'll look into a better way of managing this, though it may make sense to shift that to a separate PR.

Reply:
I've got a branch of PR feedback I'll merge in after this initial round of feedback that switches to a separate library for the generated protos definitions, and that project will pull the protos from `dapr/dapr` like the other SDKs.